apps/09todo-jquery/script.js

```diff
@@ -10,13 +11,16 @@ $(document).ready(function() {
     }
 
     var todoText = $(this).find('#todo').val();
-    $('#todo-list').append('<li><a class="done" href="#">Done</a><span class="todo-text">' + todoText + '</span></li>');
-    $('#todo-list').sortable();
+    $('#todo-list').append('<li><button class="done" href="#">Done</button><span class="todo-text">' + todoText + '</span></li>');
```

Comment:
What's the error? I never actually ran the tests (not sure how), but I definitely don't get any errors when I run in browser and everything works as expected.